careers/README.md

```diff
@@ -0,0 +1,53 @@
+# DVELP Careers
+
+We're looking to expand our team of talented people, and want to be as open about the process and opportunities as possible - hence our having all our careers information on GitHub.
+
+## Open Opportunities
+
+We're currently looking to recruit the following roles:
+
+- [Senior Fullstack Node Engineer](jobs/senior-fullstack-node-engineer.md)
+
+## Hiring Process
+
+### Apply
+
+If you want to apply for any of the positions, please email [careers@dvelp.co.uk](mailto:careers@dvelp.co.uk) with your CV - we can't wait to hear from you!
```

Comment:
Should we not be linking them to GreenHouse?

Reply:
Ooh I've just found the auto-gen'd email address for greenhouse - cool!

Reply:
Scratch that, it appears to only be for internal greenhouse usage only... is there a Greenhouse link/email we can use? Or do we throw them to the DVELP careers form on the website?